runtime/doc/starting.txt

```diff
@@ -1077,31 +1088,36 @@ giving the mapping.
 
 Defaults without a .vimrc file ~
 							*defaults.vim* *E1187*
-If Vim is started normally and no user vimrc file is found, the
-$VIMRUNTIME/defaults.vim script is loaded.  This will set 'compatible' off,
-switch on syntax highlighting and a few more things.  See the script for
-details.  NOTE: this is done since Vim 8.0, not in Vim 7.4. (it was added in
-patch 7.4.2111 to be exact).
-
-This should work well for new Vim users.  If you create your own .vimrc, it is
-recommended to add these lines somewhere near the top: >
+If Vim is started normally, the $VIMRUNTIME/defaults.vim script is loaded.
+This will set 'compatible' off, switch on syntax highlighting and a few more
+things.  See the script for details.  NOTE: this is done since Vim 8.0, not in
+Vim 7.4. (it was added in patch 7.4.2111 to be exact).
+
+This should work well for new Vim users.  If you create your own .vimrc,
+defaults.vim will (by default) not be loaded.  If you want to have the
+defaults.vim always be loaded you can use the |--load-defaults| argument or
+you can explicitly source defaults.vim by adding the following lines at the
+top of your personal vimrc file: >
+
 	unlet! skip_defaults_vim
 	source $VIMRUNTIME/defaults.vim
-Then Vim works like before you had a .vimrc.
-Copying $VIMRUNTIME/vimrc_example.vim to your .vimrc is another way to do
-this. Alternatively, you can copy defaults.vim to your .vimrc and modify it
-(but then you won't get updates when it changes).
-
-If you don't like some of the defaults, you can still source defaults.vim and
-revert individual settings.  See the defaults.vim file for hints on how to
-revert each item.
+<
+See the defaults.vim file for hints on how to revert each item and see
+|:scriptnames| to verify which runtime files have been loaded.
+
+Note: since patch 9.1.XXXX Vim can be configured to force loading the
+|defaults.vim| runtime file as the very first initialization file using the
+|--load-defaults| option.  This is done, so the effects can be undo in the
+system and/or vimrc runtime file.  To prevent loading the |defaults.vim|
+runtime file, use the |--no-load-defaults| argument (or define the environment
+variable `$VIM_NO_SOURCE_DEFAULTS`).
```

Comment:
For coherence either `VIM_NO_LOAD_DEFAULTS` or `no-source-defaults`?

Reply:
And also `$VIM_LOAD_DEFAULTS` for symmetry (and to allow tools that start `vim` to partake in the default goodness).